client/src/components/LocalEmergencyBanner/emergency-numbers-by-code.js

```diff
@@ -0,0 +1,1406 @@
+export default {
+  AD: {
+    Country: "Andorra",
+    Police: 110,
+    Ambulance: 116,
+    Fire: 118,
+  },
+  AE: {
+    Country: "United Arab Emirates",
+    Police: 999,
+    Ambulance: 999,
+    Fire: 999,
+  },
+  AF: {
+    Country: "Afghanistan",
+    Police: 119,
+    Ambulance: 112,
+    Fire: 119,
+  },
+  AG: {
+    Country: "Antigua and Barbuda",
+    Police: "911/999",
+    Ambulance: "911/999",
+    Fire: "911/999",
+  },
+  AI: {
+    Country: "Anguilla",
+    Police: 911,
+    Ambulance: 911,
+    Fire: 911,
+  },
+  AL: {
+    Country: "Albania",
+    Police: 129,
+    Ambulance: 127,
+    Fire: 128,
+  },
+  AM: {
+    Country: "Armenia",
+    Police: "112/911",
+    Ambulance: "112/911",
+    Fire: "112/911",
+  },
+  AO: {
+    Country: "Angola",
+    Police: 113,
+    Ambulance: "112/116",
+    Fire: 115,
+  },
+  AQ: {
+    Country: "Antarctica",
+    Police: 911,
+    Ambulance: 911,
+    Fire: 911,
+  },
+  AR: {
+    Country: "Argentina",
+    Police: 911,
+    Ambulance: 911,
+    Fire: 911,
+  },
+  AS: {
+    Country: "American Samoa",
+    Police: 911,
+    Ambulance: 911,
+    Fire: 911,
+  },
+  AT: {
+    Country: "Austria",
+    Police: "112/133",
+    Ambulance: 144,
+    Fire: 122,
+  },
+  AU: {
+    Country: "Australia",
+    Police: 0,
```

Comment:
```suggestion
    Police: "000",
```

As per the other comment I think we should have the actual full emergency "number" for all these. 

The other suggestion of changing all to strings - if you can do it relatively quickly great! But it's optional.

Reply:
@joshmorel Thanks for clarifying. I've reverted those changes back to the previous set up